protocol.md

```diff
@@ -1,58 +1,70 @@
 Magic Crystal
 =============
 
-HOWTO use magic-wormhole as a transport mechannism for Dark Crystal
-shards.
+_SB: Is this doc supposed to be a how-to guide for a (tech-savvy) user, or more a specification for a programmer to build an app?_
+
```

Comment:
SH: I would say both. The executive summary should give a tech savvy user a clear understanding of what the doc covers but the real content/how to should be for a programmer